chains/ethereum/listener.go

```diff
@@ -160,6 +160,15 @@ func (l *listener) getDepositEventsForBlock(latestBlock *big.Int) error {
 	l.log.Debug("Querying block for deposit events", "block", latestBlock)
 	query := buildQuery(l.cfg.bridgeContract, utils.Deposit, latestBlock, latestBlock)
 
+	mainChainId, err := l.conn.Client().ChainID(context.Background())
```

Comment:
Let's do this after the `FilterLogs` call below to be extra sure.

Reply:
Sounds good